torch/csrc/autograd/VariableTypeManual.cpp

```diff
@@ -333,7 +333,7 @@ namespace ADInplaceOrView {
   Tensor _fw_primal(c10::DispatchKeySet ks, const Tensor & self, int64_t level) {
     auto tmp = ([&]() {
       at::AutoDispatchBelowADInplaceOrView guard;
-      return at::alias(self);
+      return at::_ops::alias::redispatch(ks & c10::after_ADInplaceOrView_keyset, self);
```

Comment:
We discussed this a while with Brian and this is not ok.
You should never redispstch if you call a different function or change any of the arguments.